0003/README.md

```diff
@@ -0,0 +1,123 @@
+---
+id: 0003
+title: Centralized Logging for Tinkerbell microservices.
+status: POC Done, Code change ready
```

Comment:
The status is not one of the accepted status please have a loot at proposal 0001 and change it

Reply:
done

Reply:
Can you double check!? I still see the old status

Reply:
I think while checking how to reference other Md files in readme.md, I think I missed pushing this change. :|